skbio/stats/ordination/_principal_coordinate_analysis.py

```diff
@@ -130,20 +171,181 @@ def pcoa(distance_matrix):
     eigvecs[:, num_positive:] = np.zeros(eigvecs[:, num_positive:].shape)
     eigvals[num_positive:] = np.zeros(eigvals[num_positive:].shape)
 
+    if method == "fsvd":
+        # Since the dimension parameter, hereafter referred to as 'd',
+        # restricts the number of eigenvalues and
```

Comment:
Same comment as before, can you remove the new line and reindent this comment block. I suggest doing a pass through the rest of the source code looking for this.